builtin/app/php/data/common/dev/Vagrantfile.tpl

```diff
@@ -62,21 +62,48 @@ if ! grep "UseDNS no" /etc/ssh/sshd_config >/dev/null; then
   oe sudo service ssh restart
 fi
 
-ol "Adding apt repositories and updating..."
+ol "Adding build dependencies and updating..."
+# taken from PHP FPM docker image
 export DEBIAN_FRONTEND=noninteractive
 oe sudo apt-get update -y
-oe sudo apt-get install -y python-software-properties software-properties-common apt-transport-https
-oe sudo add-apt-repository -y ppa:ondrej/php5-{{ php_version }}
-# Seems to be required to prevent "unauthenticated packages"
-# errors out of apt-get install.
-oe sudo apt-key update
+# persistent / runtime deps
+oe sudo apt-get install -y ca-certificates curl librecode0 libsqlite3-0 libxml2
+# phpize deps
+oe sudo apt-get install -y autoconf file g++ gcc libc-dev make pkg-config re2c
+oe mkdir -p /usr/local/etc/php/conf.d
+oe gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "0BD78B5F97500D450838F95DFE857D9A90D90EC1"
+oe gpg --keyserver ha.pool.sks-keyservers.net --recv-keys "6E4F6AB321FDC07F2C332E3AC2BF0BC433CFC8B3"
+oe sudo apt-get install -y libcurl4-openssl-dev libreadline6-dev librecode-dev libsqlite3-dev libssl-dev libxml2-dev xz-utils
+oe curl -SL "http://php.net/get/php-$PHP_VERSION.tar.xz/from/this/mirror" -o php.tar.xz
+oe curl -SL "http://php.net/get/php-$PHP_VERSION.tar.xz.asc/from/this/mirror" -o php.tar.xz.asc
+oe gpg --verify php.tar.xz.asc
+oe mkdir -p /usr/src/php
+oe tar -xof php.tar.xz -C /usr/src/php --strip-components=1
+oe rm php.tar.xz*
+
+ol "Building PHP-FPM..."
```

Comment:
Why are you *building* PHP, instead of just installing from Apt? The PPA that was previously in use has a php5-fpm package.